common/src/main/cpp/extensions/openxr_fb_composition_layer_secure_content_extension_wrapper.cpp

```diff
@@ -0,0 +1,135 @@
+/**************************************************************************/
+/*  openxr_fb_composition_layer_secure_content_extension_wrapper.cpp      */
+/**************************************************************************/
+/*                       This file is part of:                            */
+/*                              GODOT XR                                  */
+/*                      https://godotengine.org                           */
+/**************************************************************************/
+/* Copyright (c) 2022-present Godot XR contributors (see CONTRIBUTORS.md) */
+/*                                                                        */
+/* Original contributed implementation:                                   */
```

Comment:
Looks like you copied the wrong license header 🙂

Reply:
Eep! I've been meaning to make a template of the `.h` and `.cpp` files, so I can copy a nice clean template, rather than copying the files from the existing extensions :-)

Reply:
Fixed in latest push